src/multivariate/mvnormal.jl

```diff
@@ -79,10 +79,10 @@ modes(d::AbstractMvNormal) = [mean(d)]
 
 function entropy(d::AbstractMvNormal)
     ldcd = logdetcov(d)
-    (length(d) * (typeof(ldcd)(log2π) + 1) + ldcd)/2
+    (length(d) * (typeof(ldcd)(log2π) + 1.) + ldcd)/2.
```

Comment:
Wasn't this fine?

Reply:
we can avoid necessary type promotions, these happen at runtime?

Reply:
but point, this is forcing everything to Float64